builds/e2e/nested-e2e.yaml

```diff
@@ -100,7 +108,7 @@ stages:
       - template: templates/nested-deploy-config.yaml
         parameters:
           deviceId: $(parentDeviceId)
-          deploymentFile: nestededge_middleLayerBaseDeployment_amqp.json
+          deploymentFile: nestededge_middleLayer_messaging_amqp.json
```

Comment:
Instead of the basic deployment file, we need to run the messaging version which contains additional test modules.